dali/aux/optical_flow/turing_of/optical_flow_turing.cc

```diff
@@ -66,13 +66,21 @@ OpticalFlowTuring::OpticalFlowTuring(dali::optical_flow::OpticalFlowParams param
   outbuf_.reset(
           new OpticalFlowBuffer(of_handle_, (width_ + 3) / 4, (height_ + 3) / 4, turing_of_,
                                 NV_OF_BUFFER_USAGE_OUTPUT, NV_OF_BUFFER_FORMAT_SHORT2));
+  if (of_params_.enable_external_hints) {
+    hintsbuf_.reset(
+            new OpticalFlowBuffer(of_handle_, (width_ + 3) / 4, (height_ + 3) / 4, turing_of_,
+                                  NV_OF_BUFFER_USAGE_HINT, NV_OF_BUFFER_FORMAT_SHORT2));
+  }
 }
 
 
 OpticalFlowTuring::~OpticalFlowTuring() {
   inbuf_.reset(nullptr);
   refbuf_.reset(nullptr);
   outbuf_.reset(nullptr);
+  if (of_params_.enable_external_hints) {
+    hintsbuf_.reset(nullptr);
```

Comment:
```suggestion
    hintsbuf_.reset();
```

Reply:
Here I followed the example in `cppreference`:
```
    std::cout << "Release and delete the owned Foo...\n";
    up.reset(nullptr); 
```
I would prefer to keep it this way